src/components/molecules/ActiveTimer.vue

```diff
@@ -17,3 +29,87 @@ import TimerStopButton from '~/atoms/TimerStopButton.vue';
 export default class ActiveTimer extends Vue {
 }
 </script>
+
+<style lang="scss" scoped>
+.ActiveTimer {
+  display: flex;
+  height: 72px;
+  background: #FFF;
+  border-radius: 8px 8px 0 0;
+  border-bottom: 1px solid #D8D8D8;
+  box-shadow: 0 -2px 8px 0px rgba(#424147, 0.17);
+  box-sizing: border-box;
+
+  &_Label {
+    flex: 1;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    height: 100%;
+    margin-left: 45px;
+    text-align: center;
+    font-size: 1.4rem;
+    overflow: hidden;
+  }
+
+  &_Time {
+    font-weight: bold;
```

Comment:
boldだと太すぎるな・・・。（boldは数値で700を指定したのと同じ太さ）
font-weightの設定をするなら500。

```suggestion
    font-weight: 500;
```